### DISCUSSION TITLE
New Format and Meeting Times

### DISCUSSION THREAD

**Discussion by MGomezN** [fatiando/community#138] [Announcements]
# :star2: Exciting Update! :star2:

## New Meeting Time
Starting next week, our meetings will be on every **Tuesday at 17:00 GMT!**

## Revolutionary Change
:tada: Introducing Group Programming Sessions to make our meetings more interactive, collaborative, and goal-driven.

:rotating_light: **Next March 5th,** we will tackle the Harmonica Issue [#423](link-to-issue) together!

:rocket: **Goals:** Interactive learning, sharing best practices, and encouraging new contributors to open their own PRs.

:iphone: Join the video call: [https://meet.jit.si/fatiando-a-terra](https://meet.jit.si/fatiando-a-terra)

:calendar: Shared Google calendar with meeting times: [https://www.fatiando.org/calendar](https://www.fatiando.org/calendar)

**Everyone is welcome!**

Meeting notes discussing this change: [https://hackmd.io/fUFzJtXtQni9VPfhUIyT5A?both#2024-02-29](https://hackmd.io/fUFzJtXtQni9VPfhUIyT5A?both#2024-02-29).

**Comment by leouieda** [comment]
Thanks for posting @MGomezN! I'm excited for the new call next week 🤩

**Comment by MGomezN** [comment]
I'm closing this discussion, to keep open the announcement with the most up to date meeting times :)
